articles/fleet-4.50.0.md

```diff
@@ -0,0 +1,138 @@
+
+For users who self-manage (host) Fleet, this feature requires connecting Fleet with an S3 bucket. See how in the server configuration reference [here](https://fleetdm.com/docs/configuration/fleet-server-configuration#s-3).
+
+## Policy description and resolutions aided by AI
+
+Fleet aims to enhance how policy descriptions and resolutions are generated for policies. This new functionality leverages artificial intelligence (AI) to automatically populate policy details directly from SQL queries that define policies. It is important to note that Fleet does not use any data to train large language models (LLMs); only the policy queries (SQL) are sent to the LLM for generating descriptions and resolutions. When administrators create or modify a policy, they can opt to have the description and resolution fields filled instantly by the AI based on the context and content of the SQL query. This process not only simplifies the task of policy creation by providing pre-generated, meaningful explanations and solutions but also ensures consistency and comprehensiveness in policy documentation. 
+
+This improvement enhances the user experience for administrators and end-users by enabling transparent communication of policy purposes and actions to end-users. This can be especially useful in scenarios like scheduled [maintenance windows](https://fleetdm.com/announcements/fleet-in-your-calendar-introducing-maintenance-windows) visible to users through calendar events or device notifications. By automating the generation of detailed, relevant policy descriptions, Fleet helps ensure that all parties understand what each policy entails and why it is important, enhancing the organization's overall security posture and compliance.
+
+
+## Mac Admins SOFA support
+
+Fleet has integrated support for the Mac Admins [SOFA](https://github.com/macadmins/sofa) (Structured Open Feed Aggregator), enhancing its capabilities to provide comprehensive tracking and surfacing of update information for macOS and iOS environments. SOFA, known for its machine-readable feed and user-friendly web interface, offers continuous updates on XProtect data, OS updates, and detailed release information. This integration within Fleet is facilitated through the recent updates to the [Mac Admins osquery extension](https://github.com/macadmins/osquery-extension), which now includes tables specifically for security release information (`sofa_security_release_info`) and unpatched CVEs (`sofa_unpatched_cves`).
```

Comment:
```suggestion
Fleet has integrated support for the Mac Admins [SOFA](https://github.com/macadmins/sofa) (Structured Open Feed Aggregator), enhancing its capabilities to provide comprehensive tracking and surfacing of update information for macOS and iOS environments. SOFA, known for its machine-readable feed and user-friendly web interface, offers continuous updates on XProtect data, OS updates, and detailed release information. This integration within Fleet is facilitated through new tables specifically for security release information ([`sofa_security_release_info`](https://fleetdm.com/tables/sofa_security_release_info)) and unpatched CVEs ([`sofa_unpatched_cves`](https://fleetdm.com/tables/sofa_unpatched_cves)).
```

Adding links to the tables on fleetdm.com/tables. 

Suggest prioritizing that over MacAdmins osquery extension. We're bundling these tables in the fleetd agent. Folks don't need to worry about the MacAdmins extension.